packages/@adobe/react-spectrum/update-readme.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+let fs = require('fs');
+let packageJson = require('./package.json');
+
+/***
```

Comment:
should we keep this up in the scripts folder we have in the root?
should be able to use import syntax if we do that too

Reply:
I had placed it here because it only really works in the context of this package, but I'm happy to follow your lead. Let me know.